src/Documents.php

```diff
@@ -62,41 +62,83 @@ private function endPointPath(string $action = ''): string
      */
     public function create(array $document): array
     {
-        return $this->apiCall->post($this->endPointPath(''), $document);
+        return $this->apiCall->post($this->endPointPath(''), $document, true);
+    }
+
+    /**
+     * @param array $document
+     *
+     * @return array
+     * @throws TypesenseClientError|GuzzleException
+     */
+    public function upsert(array $document): array
+    {
+        return $this->apiCall->post($this->endPointPath(''), $document, true, ['action' => 'upsert']);
+    }
+
+    /**
+     * @param array $document
+     *
+     * @return array
+     * @throws TypesenseClientError|GuzzleException
+     */
+    public function update(array $document): array
+    {
+        return $this->apiCall->post($this->endPointPath(''), $document, true, ['action' => 'update']);
     }
 
     /**
      * @param array $documents
+     * @param array $options
      *
      * @return array
      * @throws TypesenseClientError|GuzzleException|\JsonException
      */
-    public function createMany(array $documents): array
+    public function createMany(array $documents, array $options = []): array
```

Comment:
I've noticed that `createMany` method will be deprecated, we need to update the laravel driver since this will brake it.
And `import` method will work instead of `createMany`?

Reply:
`createMany` is deprecated, but not removed in this version - it will just call `import` internally. So this version of the PHP library will not break backward compatibility.

> And import method will work instead of createMany?

Correct. If you just change `createMany` to `import` it should work with no additional changes.